OpenRA.Mods.Common/Traits/Conditions/ToggleChargedConditionOnOrder.cs

```diff
@@ -46,6 +46,9 @@ public class GrantConditionOnDeployWithChargeInfo : PausableConditionalTraitInfo
 		[Desc("Can " + nameof(DeployedCondition) + " be canceled by followup deploy order?")]
 		public readonly bool CanCancelCondition = false;
 
+		[Desc("Should we interrupt the current activity and wait for it to finish?")]
```

Comment:
```suggestion
		[Desc("Should we interrupt the current activity if the toggle order is not queued?")]
```

Reply:
my version descried the behaviour correctly

Reply:
I don't understand "interrupt and wait to finish". The point is to _not_ wait for it to finish.
Also that depends on whether the order is queued 🤷‍♂️

Reply:
You interrupt but the activity might still take quite a few frames to finish. We wait for it, instead of skipping like the default behaviour in this PR is